bulk_grades/api.py

```diff
@@ -282,10 +293,14 @@ def __init__(self, **kwargs):
             filter_assignment_type=kwargs.get('assignment_type', None),
         )
         self.append_columns(
-            self._subsection_column_names(self._subsections.keys(), self.subsection_prefixes)
+            self._subsection_column_names(
+                self._subsections.keys(),  # pylint: disable=dict-keys-not-iterating, useless-suppression
```

Comment:
doesn't the `useless-suppression` tell you that the `dict-keys-not-iterating` is...useless?

Reply:
Yes. When I have nothing there, it complains about `dict-keys-not-iterating`. When I put it there it complains about `useless-suppression ` 🤷‍♂